### PR TITLE
Fixing #53 #38, allows deleting tokens with backspaces when user has a custom keyboard

### PR DESCRIPTION
**Pull request by frobichaud**

### CODE REVIEW

**Comment by frobichaud** [comment]
@ayanonagon, can you please review and merge this?

File: VENTokenField/VENTokenField.m [hunk 1]
```diff
@@ -534,7 +534,14 @@ - (void)textFieldDidBeginEditing:(UITextField *)textField
 
 - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRange)range replacementString:(NSString *)string
 {
-    [self unhighlightAllTokens];
+    BOOL isPressedBackspaceAfterSingleSpaceSymbol = [textField.text isEqualToString:@""] && range.location == 0 && range.length == 0 && string.length == 0;
```
Comment (ayanonagon) [inline VENTokenField/VENTokenField.m]: `pressedBackspaceAfterSingleSpace` would be cleaner I think! :octocat:

**Comment by ayanonagon** [comment]
1 tiny fix, then we can merge. Thank you for the pull-request! :balloon:

File: VENTokenField/VENTokenField.m (hunk 1 above)
Comment (StarWars) [reply VENTokenField/VENTokenField.m:537]: @frobichaud It is not working with SwiftKey keyboard

**Review by StarWars — CHANGES_REQUESTED** [review]
The proposed solution doesn't work with the current version of SwiftKey keyboard.

**Pull request closed without merging**